### PR TITLE
Update scripts to node, remove wiki commit from script

### LINKED ISSUE #18: Ignore wiki articles in repo

### LINKED ISSUE #45: wiki-update script must not try to commit to history

### LINKED ISSUE #46: Submodules are not working correctly

### LINKED ISSUE #47: Change reference to iojs in scripts to node

### PR DESCRIPTION
**Pull request by ltegman**
Also ignores data/wiki. Closes #47, closes #45, closes #18, closes #46

### CODE REVIEW

File: bin/wiki-update.sh [hunk 1]
```diff
@@ -9,9 +9,3 @@ cd ../..
 rm data/wiki/*
 
 cp data/fcc.wiki/*md data/wiki
```
Comment (abhisekp) [inline bin/wiki-update.sh]: `cp data/fcc.wiki/*.md data/wiki/`

**Comment by abhisekp** [comment]
@ltegman have you tested this using demobot. Last time i removed the `/data/wiki` directory camperbot crashed using the `update` command.

File: bin/run.sh [hunk 2]
```diff
@@ -13,8 +13,8 @@
 #LOG_LEVEL=10 \
 #PORT=7891 \
 
-# nodemon -x --harmony iojs app.js
+# nodemon -x node app.js
 
-    # nodemon --harmony app.js
+    # nodemon app.js
```
Comment (abhisekp) [inline bin/run.sh]: I think these comments are redundant.

File: bin/wiki-update.sh (hunk 1 above)
Comment (ltegman) [inline bin/wiki-update.sh]: This line hasn't been changed by this PR.

**Comment by ltegman** [comment]
@abhisekp You need a /data/wiki folder, but we shouldn't be tracking it in repo.

File: bin/run.sh (hunk 2 above)
Comment (ltegman) [inline bin/run.sh]: These are commented out. I'm leaving this scripts as they were apart from changing to reflect that we are using node 4.2.X now.

**Comment by abhisekp** [comment]
@ltegman ignoring the `/data/wiki` doesn't do anything since it has already been added to the repo. To completely remove the wiki pages, the pages have to be removed from repo using `git rm`.
I think there is nothing wrong in keeping a blank `wiki` directory since the wiki pages will be fetched from there.

**Comment by ltegman** [comment]
@abhisekp @BerkeleyTrue I think this should be ready now. I just went ahead and removed the whole need for a copy of the wiki outside of the submodule. Worked in my local testing.

**Comment by SaintPeter** [comment]
Note:  You may need to do a `git submodule init` and a `git submodule update` to pull down the latest content.  Then you might have to checkout the right branch.  That's what I needed to do when I used it for the wiki-generation scripts.

**Comment by ltegman** [comment]
@SaintPeter Updated the README to reflect that you need to init the submodules to get the bot working.

**Comment by SaintPeter** [comment]
I'm confused:  Which folder will actually be linked to the FCC wiki?
Also, what the heck is with the "kbase.wiki.git"?  Does that need to stay?

**Comment by ltegman** [comment]
Removed kbase.wiki since it was not used anywhere in the codebase. And the folder wikis will be linked to and used from now is the data/fcc.wiki submodule, no more copying into data/wiki :smile:

**Comment by SaintPeter** [comment]
@ltegman and I QAed this together today.  To the best of my ability to test, it appears to be good to go.

I'm going to merge.

**Pull request merged**